files/openmw.appdata.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2015 Alexandre Moine <nobrakal@gmail.com> -->
+<application>
+ <id type="desktop">openmw.desktop</id>
+ <metadata_license>CC0-1.0</metadata_license>
```

Comment:
I think that depends on jurisdiction. Some places don't allow you to specifically place something in the public domain, and instead you have to just use a very liberal license (which I presume what CC0 is; a copyright license that imposes as few restrictions as possible). In such a case, it still needs proper copyright attribution otherwise it's assumed All Rights Reserved.